CRM/Odoosync/Mail/Error.php

```diff
@@ -0,0 +1,251 @@
+<?php
+
+/**
+ * This class sends error messages to emails from Odoo settings
+ */
+class CRM_Odoosync_Mail_Error {
+
+  /**
+   * Default domain email name
+   *
+   * @var array
+   */
+  const DEFAULT_DOMAIN_EMAIL_NAME = "CiviCRM";
+
+  /**
+   * Log messages
+   *
+   * @var array
+   */
+  private $log = [];
+
+  /**
+   * Recipients' emails use for send error messages
+   *
+   * @var array
+   */
+  private $emails = [];
+
+  /**
+   * Error message
+   *
+   * @var string
+   */
+  private $errorMessage;
+
+  /**
+   * Entity type
+   *
+   * @var string
+   */
+  private $entityType;
+
+  /**
+   * Entity id
+   *
+   * @var string
+   */
+  private $entityId;
+
+  /**
+   * Domain email name
+   *
+   * @var string
+   */
+  private $domainEmailName;
+
+  /**
+   * Domain email address
+   *
+   * @var string
+   */
+  private $domainEmailAddress;
```

Comment:
and this to senderEmailAddress, actually rename any occurance for domainEmail with sendEmail in this class which includes setter methods, variable names ..etc